website/publish/domain_name.rst

```diff
@@ -81,35 +81,42 @@ We can now apply the redirection from your domain name's manager account:
    - `Namecheap <https://www.namecheap.com/support/knowledgebase/article.aspx/9646/10/how-can-i-set-up-a-cname-record-for-my-domain>`__
    - `OVH <https://www.ovh.co.uk/g1519.exchange_20132016_how_to_add_a_cname_record>`__
 
-How to enable SSL (HTTPS) for my Odoo instance
+How To Enable SSL (HTTPS) For My Odoo Database
 ==============================================
 
-Until recently, Odoo users needed to use a third-party CDN service provider such as CloudFlare to enable SSL.
+Until recently, Odoo users needed to use a third-party CDN service provider (such as CloudFlare) 
+to enable SSL. This is not required anymore.
 
-It is not required anymore: Odoo generates the certificate for you automatically, using integration with `Let's Encrypt Certificate Authority and ACME protocol <https://letsencrypt.org/how-it-works/>`__.
+Odoo generates the certificate for you automatically using integration with 
+`Let's Encrypt Certificate Authority and ACME protocol <https://letsencrypt.org/how-it-works/>`__.
```

Comment:
Please do as per https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_guidelines.html#use-single-underscore-suffixes-for-hyperlink-references

Reply:
I believe I have fixed the single underscore issue.